lib/rucio/cli/account.py

```diff
@@ -144,13 +144,13 @@ def limit():
 @limit.command("list", help="Shows the space used, the quota limit and the quota left for an account for every RSE where the user have quota.")
 @click.argument("account-name")
 @click.option("--rse", "--rse-name", help="Show usage for only for this RSE.")
+@click.option("--unique", is_flag=True, default=False, help="Count unique replicas to avoid double-counting when multiple locks exist.")
 @click.pass_context
-def limit_list(ctx, account_name, rse):
+def limit_list(ctx, account_name, rse, unique):
```

Comment:
The autotest is failing due to missing type annotations. I noticed other functions in this module also lack them. Should I add the annotations, or is there a reason CLI modules are exempted?

```suggestion
def limit_list(ctx: click.Context, account_name: str, rse: Optional[str], unique: bool) -> None:
```

Reply:
Yeah, this is a discussion we need to have in full - see https://github.com/rucio/rucio/issues/7582